prisma/schema.prisma

```diff
@@ -67,3 +68,20 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+// Profile schema
+model Profile {
+    id            Int     @id @default(autoincrement())
+    user          User    @relation(fields: [userId], references: [id])
+    userId        String  @unique
+    preferredName String?
+    phoneNumber   String?
+    bio           String?
+    role          UserRole @relation(fields: [roleId], references: [id])
+    roleId        String @default("MENTEE")
```

Comment:
Do we need a default value here? I think it is better to not have one at all. The role is pretty important and we should explicitly specify it on creation.